docs/index.html

```diff
@@ -690,7 +694,11 @@ <h1 class="display-4 theme-content-element-title">Cards</h1>
 		<span class="theme-content-element-source-label">Source:</span>
 		<span class="theme-content-element-source-path">cards.md</span>
 	</span>
-	<div class="theme-content-element-description"><p>In component based services (especially internal systems), content can be structured using Cards.</p>
+	<div class="theme-content-element-description"><p>A card is composed of several elements (eg title, description, icon, illustration). Combined, these elements are related to a single theme or to a destination. A card does not have an active / selected state. Cards contain interactive content, like function or other action points. Since cards are the starting point for more detailed info, they should contain a limited number of actions.</p>
+<h3>✅ When to use:</h3><p>Cards are used when you want to view content related to a theme. The text of the card is recommended to be kept at a maximum of 70 characters per line to improve the readability for the user. The content should be short and kept to a minimum. Padding inside the card is recommended to have spacing-3. If there are multiple cards in a row, the cards should have the same height. </p>
```

Comment:
padding inne i korter, er ikke det noe vi styrer?

Reply:
Jo. Däremot så kanske finns behov för att bryta designsystemet någon gång. Då kan det vara bra att det åtminstone står hur det borde vara

Reply:
Er da denne kommentaren ment for designer, og ikke utvikler?

Reply:
Isåfall er jo spørsmålet om designere skal ha en egen dokumentasjon å se på? Det kan jo bli en del mer arbeid om man skal skille dette?

Reply:
Vi kan skriva att det är anbefallt om det är så att de inte har en designer på teamet. Men ja, all padding och liknande borde komma från design och så ska utveckling freestyla så lite som möjligt egentligen. Kanske räcker att det står att det är anbefallt och så kan man styra det själv som utvikler?

Reply:
Kan hende vi trenger på noen elementer en egen linje for designer også 
Tror vi må også definere hva kort er for oss i designsystemet

Reply:
Enig at det kan trenges en definisjon på hva kort er ment for spesifikt i våre systemer. Kom gjerne med innspill på hva som skal skrives her!

Kan endre denne delen til:

 ✅ When to use: 
Cards are used when you want to view content related to a theme. The text of the card is recommended to be kept at a maximum of 70 characters per line to improve the readability for the user. The content should be short and kept to a minimum. A header-icon in the top left corner can be used to improve the recognition of content. A card can also have an icon prompting the user to go to settings for this specific content. 

For designer: 
Padding inside the card is recommended to have spacing-3. If there are multiple cards in a row, the cards should have the same height. The icon in the label should be inside a circle, aligned to the H1 element. The icon should clearly be connected to the title, and be relevant to content. If a card is clickable, this should be displayed with a light border, view more and an arrow.

Reply:
Liker det. Kan også bruke denne på "For designer:" https://emojipedia.org/lower-left-fountain-pen/

Reply:
A card should only encapsulate one type of content.